src/components/SharedComponents/StyledRegionName/StyledRegionName.tsx

```diff
@@ -32,7 +32,9 @@ const StyledRegionName: React.FC<{
       {regionNameMain}
       {regionSuffix && (
         <Suffix>
-          {regionSuffix !== 'metro' ? ` ${regionSuffix}` : ` ${regionSuffix},`}
+          {condensed && regionSuffix !== 'metro'
+            ? ` ${regionSuffix}`
+            : ` ${regionSuffix},`}
```

Comment:
This works for now, but maybe we should split the rendering methods by region type to make this easier to read at some point? (also, maybe we should add a `suffix` attribute to the `Region` objects)